### DISCUSSION TITLE
Show recordings as video preview

### DISCUSSION THREAD

**Discussion by staabm** [RecordReplay/devtools#4144] [Ideas]
It would be cool if there would be some way to share a recording as a preview video. 

E.g. imagine a dev putting together a PR and adding a replay link into the description. People reviewing the PR could immediately „watch“ the recording without even jumping into the replay website to grasp what the recording is about.

in case people want to drill in further they could still click the link and jump into the rr debugging experience

**Comment by jasonLaster** [comment]
Thanks for filing @staabm. This is a great idea and worth exploring.

Previously, I had been thinking about this in terms of a replay's url that would be added to a PR, but this has gotten me thinking :)

Why not make it easy to share a url for the video mp4 file? Why not make it easy to share a url for the current screen? Why not make it easy to crop a screen or trim a video segment?


Yesterday I filed this [issue](https://github.com/RecordReplay/devtools/issues/4119). I think if we design this right, things like this would be easier...

**Comment by jonbell-lot23** [comment]
March 15, 2022:

We've talked about sharing video over the past few months. Any status to share?
